lms/static/sass/multicourse/_course_about.scss

```diff
@@ -1,6 +1,11 @@
 .course-info {
   .container {
     padding-bottom: 120px;
+
+    @media screen and (max-width: $bp-screen-md) {
```

Comment:
For screen adjustments we now use the bootstrap standards of @include media-breakpoint-down(sm) { ... } (that sets that styling for all screens that are small or extra small).

See https://v4-alpha.getbootstrap.com/layout/overview/